pandas/tests/tseries/offsets/test_business_quarter.py

```diff
@@ -169,7 +173,7 @@ def test_offset(self, case):
 
 
 class TestBQuarterEnd(Base):
-    _offset = BQuarterEnd
+    _offset: type[QuarterOffset] = BQuarterEnd
```

Comment:
not a problem, but im surprised all these are needed.  what breaks without them?

Reply:
Thank you for your review! Will probably take me a bit to address them :)

`_offset` is defined in the class `Base` as a super-type of `QuarterOffset`. The inherited tests call `__init__` (some of them can take more arguments) and call additional methods on those objects.